### PR TITLE
fix(ldp): data gathering tools

### PR DESCRIPTION
**Pull request by pdepaepe**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix OB-5196
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

This will fix the data gathering tool section (edition, configuration) of Logs Data Platform control panel.

### CODE REVIEW

File: packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.html [hunk 1]
```diff
@@ -145,13 +145,11 @@
...
         <p data-translate="inputs_logs_configure_logstash_desc2"></p>
-        <oui-spinner
-            data-ng-if="ctrl.input.loading || ctrl.loading.engine"
-        ></oui-spinner>
+        <oui-spinner data-ng-if="ctrl.input.loading"></oui-spinner>
         <form
             novalidate
             name="ctrl.logstashForm"
-            data-ng-if="!(ctrl.input.loading || ctrl.loading.engine)"
+            data-ng-if="!(ctrl.input.loading)"
```
Comment (ghyenne) [inline packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.html:152]:
```suggestion
            data-ng-if="!ctrl.input.loading"
```

File: packages/manager/modules/dbaas-logs/src/logs/detail/inputs/logs-inputs.service.js [hunk 2]
```diff
@@ -439,14 +491,28 @@ export default class LogsInputsService {
...
+  }
+
+  getFlowggerConfiguration(serviceName, input) {
+    return this.$http.get(
+      `/dbaas/logs/${serviceName}/input/${input.inputId}/configuration/flowgger`,
+    );
+  }
+
+  updateLogstash(serviceName, input) {
+    console.log(input);
```
Comment (ghyenne) [inline packages/manager/modules/dbaas-logs/src/logs/detail/inputs/logs-inputs.service.js:507]:
```suggestion

```

File: packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js [hunk 3]
```diff
@@ -64,59 +53,36 @@ export default class LogsInputsAddConfigureCtrl {
...
-      configuration.filterSection,
-    );
-    this.configuration.logstash.patternSection = this.CucControllerHelper.constructor.htmlDecode(
-      configuration.patternSection,
-    );
+    if (str) {
+      const lines = str.split(/\r\n|\r|\n/);
+      return lines.length;
+    }
+    return 5;
```
Comment (ghyenne) [inline packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js:69]: Maybe you can move in constant file ?

Comment (pdepaepe) [reply packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js:69]: It's a static function so access to LogsConstants looks for me complicate. What is your recommendation?

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**